### PR TITLE
Fix cli plugin uninstall

### LINKED ISSUE #3228: CLI plugin uninstall: NoMethodError : undefined method `name' for "aws":String
```
$ DEBUG=true kontena plugin uninstall aws
DEBUG Kontena CLI 1.5.0.dev (ruby-2.3.1+x86_64-linux-gnu)
DEBUG Loading plugin kontena-plugin-app-command from /home/terom/.kontena/gems/2.3.1/gems/kontena-plugin-app-command-0.1.0/lib/kontena_cli_plugin.rb
DEBUG Loading plugin kontena-plugin-aws from /home/terom/.kontena/gems/2.3.1/gems/kontena-plugin-aws-0.3.1/lib/kontena_cli_plugin.rb
DEBUG Loading plugin kontena-plugin-digitalocean from /home/terom/.kontena/gems/2.3.1/gems/kontena-plugin-digitalocean-0.3.3/lib/kontena_cli_plugin.rb
DEBUG Running Kontena::MainCommand with ["plugin", "uninstall", "aws"] -- callback matcher = 'nil'
DEBUG Running Kontena::Cli::PluginCommand with ["uninstall", "aws"] -- callback matcher = 'nil'
DEBUG Running Kontena::Cli::Plugins::UninstallCommand with ["aws"] -- callback matcher = 'plugins uninstall'
DEBUG Execution took 0.113 seconds
DEBUG NoMethodError
/home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/plugin_manager/common.rb:38:in `block in installed': undefined method `name' for "aws":String (NoMethodError)
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/plugin_manager/common.rb:38:in `each'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/plugin_manager/common.rb:38:in `find'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/plugin_manager/common.rb:38:in `installed'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/plugin_manager/common.rb:43:in `installed?'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/cli/plugins/uninstall_command.rb:13:in `block in execute'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/cli/plugins/uninstall_command.rb:12:in `each'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/cli/plugins/uninstall_command.rb:12:in `execute'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/command.rb:215:in `run'
	from /home/terom/.gem/ruby/2.3.0/gems/clamp-1.1.2/lib/clamp/subcommand/execution.rb:11:in `execute'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/command.rb:215:in `run'
	from /home/terom/.gem/ruby/2.3.0/gems/clamp-1.1.2/lib/clamp/subcommand/execution.rb:11:in `execute'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/lib/kontena/command.rb:215:in `run'
	from /home/terom/.gem/ruby/2.3.0/gems/clamp-1.1.2/lib/clamp/command.rb:132:in `run'
	from /home/terom/.gem/ruby/2.3.0/gems/kontena-cli-1.5.0.dev/bin/kontena:19:in `<top (required)>'
	from /home/terom/.local/bin/kontena:23:in `load'
	from /home/terom/.local/bin/kontena:23:in `<main>'
```

### PR DESCRIPTION
**Pull request by SpComb**
Fixes #3228 to remove the `Kontena::PluginManager::Common` mixin.

No specs because mocking out `installed?` wouldn't really help catch this issue, so rely on the e2e specs instead.

### CODE REVIEW

File: cli/lib/kontena/cli/plugins/uninstall_command.rb [hunk 1]
```diff
@@ -6,10 +6,10 @@ class UninstallCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::PluginManager::Common
 
-    parameter 'NAME ...', 'Plugin name', attribute_name: :plugins
+    parameter 'NAME ...', 'Plugin name', attribute_name: :plugins_to_uninstall
 
     def execute
-      plugins.each do |name|
+      plugins_to_uninstall.each do |name|
```
Comment (kke) [inline cli/lib/kontena/cli/plugins/uninstall_command.rb]: Hmm, shouldn't that have been `plugins_list` anyway?

Comment (SpComb) [reply cli/lib/kontena/cli/plugins/uninstall_command.rb]: Changed to stick to the standard clamp `name_list` convention.

**Comment by SpComb** [comment]
While waiting for travis to run; verified to fix the e2e specs locally:

```
plugin upgrade
  upgrades all plugins
  does nothing if no updates

plugin uninstall
  removes installed plugin

Top 3 slowest examples (15.85 seconds, 100.0% of total time):
  plugin upgrade upgrades all plugins
    8.32 seconds ./spec/features/plugin/upgrade_spec.rb:16
  plugin upgrade does nothing if no updates
    6.08 seconds ./spec/features/plugin/upgrade_spec.rb:32
  plugin uninstall removes installed plugin
    1.45 seconds ./spec/features/plugin/uninstall_spec.rb:8


Finished in 15.85 seconds (files took 0.09373 seconds to load)
3 examples, 0 failures
```

**Comment by SpComb** [comment]
And also tested manually:
```
$ kontena plugin list
NAME          VERSION   DESCRIPTION
app-command   0.1.0     Restores the "kontena app" subcommand back to Kontena CLI v1.4+
$ kontena plugin uninstall est
 [error] Plugin est has not been installed
$ kontena plugin uninstall app-command
 [done] Uninstalling plugin app-command     
$ kontena plugin list
NAME  VERSION  DESCRIPTION
```

**Pull request merged**